src/core/reopt.jl

```diff
@@ -66,10 +94,21 @@ end
 Solve the model using a `Scenario` or `BAUScenario`.
 """
 function run_reopt(m::JuMP.AbstractModel, s::AbstractScenario)
-	if s.site.CO2_emissions_reduction_min_fraction > 0.0 || s.site.CO2_emissions_reduction_max_fraction < 1.0
-		error("To constrain CO2 emissions reduction min or max percentages, the optimal and business as usual scenarios must be run in parallel. Use a version of run_reopt() that takes an array of two models.")
+	
+	instantiate_logger() # have to instantiate here to capture the CO2 emission reduction error below, can remove otherwise
```

Comment:
instantiate_logger is called in the Scenario constructor and a scenario is an argument to this function, so wouldn't it have already been called?

Reply:
Yes, thanks for catching that. Makes sense to me as well. I have tested and removed this extra logger instantiation in https://github.com/NREL/REopt.jl/pull/118/commits/eb1e50983b7dd347f801a7911c1c0dd87b3d581e.